### PR TITLE
Add packaging

### PR DESCRIPTION
**Pull request by Vampouille**
This PR adds:
* setup.py and requirements.txt with dependencies and description
* debian folder to build a debian package

The debian package target debian buster

### CODE REVIEW

**Comment by Vampouille** [comment]
@dvzrv This PR include a valid setup.py

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: End Users/Desktop",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3.7",
+        "Topic :: Multimedia :: Sound/Audio :: Capture/Recording",
+        "Topic :: Multimedia :: Sound/Audio :: MIDI",
+    ],
+        # C'est un système de plugin, mais on s'en sert presque exclusivement
```
Comment (dvzrv) [inline setup.py]: Would be great to add these comments in english

File: debian/control [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Source: superboucle
+Maintainer: Julien Acroute <superboucle@nura.eu>
+Section: sound
+Priority: optional
+Standards-Version: 3.9.4
+Build-Depends: debhelper (>= 9), dh-python,
+ python3-all (>= 3.3), python3-setuptools, python3-soundfile (>= 0.10),
+ python3-cffi (>= 1.12), python3-numpy (>= 1.16), python3-pyqt5 (>= 5.11)
+
```
Comment (trebmuh) [inline debian/control]: Please, one line by item + alphabetically sorted

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Package: superboucle
+Architecture: all
+Depends: ${shlibs:Depends}, ${misc:Depends},
+ python3-soundfile (>= 0.10), python3-cffi (>= 1.12), python3-numpy (>= 1.16),
+ python3-pyqt5 (>= 5.11), libsndfile1-dev, libjack-jackd2-dev
```
Comment (trebmuh) [inline debian/control]: same here

File: debian/control [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+Description: SuperBoucle is a loop based software fully controllable with any
```
Comment (trebmuh) [inline debian/control]: You need to start with a "short description" (such as "loop based software midi device controllable"), then line break, then a full description

**Comment by trebmuh** [comment]
@Vampouille note that you can merge it as is and I'll PR deb improvements afterward. Probably easier that way.

**Comment by trebmuh** [comment]
@Vampouille note that you can merge the Debian packaging as-is and I'll PR improvements afterward. Probably easier that way.

File: setup.py (hunk 1 above)
Comment (Vampouille) [reply setup.py]: I will remove it because it came from an example

**Comment by trebmuh** [comment]
:+1:

**Comment by Vampouille** [comment]
> missing a "," at the end of the line

Fixed with 6067b26

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**